detection/notebooks/11_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<p>we don't use the parameter "experiment_name" beyond here. We could just define it in the Experiment constructor instead of setting a parameter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Good point. Changed.